### ISSUE TITLE
[BUG] v3 `update` ttl to `undefined` fails

### ISSUE THREAD

**Issue by fishcharlie** [dynamoose/dynamoose#1475]
### Summary:

It seems like doing an `update` call in v3 and setting the `ttl` to `undefined`, fails.

#### Schema
```
// Code here
```

#### Model
```
// Code here
```

#### General
```
// Code here
```

### Environment:

Operating System:
Operating System Version:
Node.js version (`node -v`):
NPM version: (`npm -v`):
Dynamoose version:

### Other:
- [ ] I have read through the Dynamoose documentation before posting this issue
- [ ] I have searched through the GitHub issues (including closed issues) and pull requests to ensure this issue has not already been raised before
- [ ] I have searched the internet and Stack Overflow to ensure this issue hasn't been raised or answered before
- [ ] I have tested the code provided and am confident it doesn't work as intended
- [ ] I have filled out all fields above
- [ ] I am running the latest version of Dynamoose

**Comment by KennethWKZ** [comment]
@fishcharlie is it this bug will fix in v3.2 for sure? As it is affected our application to use update table

**Comment by fishcharlie** [comment]
@KennethWKZ No. Not guaranteed. If someone makes a PR for it I'll happily review and get it into a release.

Otherwise I try to prioritize things based on 👍 emojis to the original issue. But I've been so busy with work recently so I'm not sure when I'll get around to it.
